docs/integration/ldap.md

```diff
@@ -0,0 +1,166 @@
+`ldap` (you can set it to `custom` if you really want to override `PULP_AUTHENTICATION_BACKENDS` variable)
+
+```bash
+PULP_AUTHENTICATION_BACKEND_PRESET=ldap
+```
+
+Specific django_auth_ldap settings
+
+!!! tip
+    depending on the LDAP server some of the following settings might need change.
+
+The following keys are **required** in order to have LDAP enabled:
+
+```bash
+PULP_AUTH_LDAP_SERVER_URI="ldap://ldap:10389"
+PULP_AUTH_LDAP_BIND_DN="cn=admin,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_BIND_PASSWORD="GoodNewsEveryone"
+PULP_AUTH_LDAP_USER_SEARCH_BASE_DN="ou=people,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_USER_SEARCH_SCOPE="SUBTREE"
+PULP_AUTH_LDAP_USER_SEARCH_FILTER="(uid=%(user)s)"
+PULP_AUTH_LDAP_GROUP_SEARCH_BASE_DN="ou=people,dc=planetexpress,dc=com"
+PULP_AUTH_LDAP_GROUP_SEARCH_SCOPE="SUBTREE"
+PULP_AUTH_LDAP_GROUP_SEARCH_FILTER = "(objectClass=Group)"
+PULP_AUTH_LDAP_GROUP_TYPE_CLASS="django_auth_ldap.config:GroupOfNamesType"
+```
+
+Optional variables:
+
+```bash
+PULP_AUTH_LDAP_USER_ATTR_MAP={first_name="givenName", last_name="sn", email="mail"}
+# NOTE: the above is formatted as a toml hashmap
+
+PULP_AUTH_LDAP_MIRROR_GROUPS=true
+# The above is what enabled group mirroring
+```
+
+You can limit which groups are mirrored if you don't want all the groups to be added do Hub.
+
+```bash
+PULP_AUTH_LDAP_MIRROR_GROUPS_EXCEPT=['foobar']
+# this syncs all groups except the `foobar`
+```
+
+Require a specific group for all users
+
+```bash
+PULP_AUTH_LDAP_REQUIRE_GROUP='hub_users'
+# Only users belonging to this group will be allowed to login
+```
+
+Mapping groups from LDAP to user attributes on Django:
+
+Ex: Users belonging to `admin_staff` on LDAP is `superuser` on Django.
+
+```bash
+PULP_AUTH_LDAP_USER_FLAGS_BY_GROUP__is_superuser="cn=admin_staff,ou=people,dc=planetexpress,dc=com"
+```
+
+And the same logic can be applied to any other attribute:
+
+```bash
+PULP_AUTH_LDAP_USER_FLAGS_BY_GROUP__is_staff="cn=ship_crew,ou=people,dc=planetexpress,dc=com"
+```
+
+
+Or optionally put on `/etc/pulp/setting.py`
+
+```python
+AUTH_LDAP_USER_FLAGS_BY_GROUP = {
+    "is_active": "cn=active,ou=groups,dc=example,dc=com",
+    "is_staff": (
+        LDAPGroupQuery("cn=staff,ou=groups,dc=example,dc=com")
+        | LDAPGroupQuery("cn=admin,ou=groups,dc=example,dc=com")
+    ),
+    "is_superuser": "cn=superuser,ou=groups,dc=example,dc=com",
+}
+```
+
+TLS verification
+
+```bash
+# Make ldap to call start_tls on connections
+PULP_AUTH_LDAP_START_TLS=true
+
+# If using self signed certificates set this
+PULP_GALAXY_LDAP_SELF_SIGNED_CERT=true
+```
+
+Logging:
+
+```bash
+# Enable LDAP logging handler
+PULP_GALAXY_LDAP_LOGGING=true
+```
+
+Cache
+
+```bash
+# Change the caching lifetime in seconds (for groups and users search)
+PULP_AUTH_LDAP_CACHE_TIMEOUT=3600
+```
+
+
+More settings can be found on https://django-auth-ldap.readthedocs.io/en/latest/reference.html#settings
```

Comment:
Would it be worthwhile to add a note about the LDAP user/group matching being case insensitive? (The blurb at the top of https://django-auth-ldap.readthedocs.io/en/latest/users.html).

I don't remember if with support case sensitive user/group names in galaxy_ng or not, so it may depend on that.